mycroft/messagebus/load_config.py

```diff
@@ -0,0 +1,34 @@
+from collections import namedtuple
+
+from mycroft.configuration import Configuration
+from mycroft.util.log import LOG
+
+MessageBusConfig = namedtuple(
+    'MessageBusConfig',
+    ['host', 'port', 'route', 'ssl']
+)
+
+
+def load_message_bus_config(**overrides):
+    LOG.info('Loading message bus configs')
+    config = Configuration.get()
+
+    try:
+        websocket_configs = config['websocket']
+    except KeyError as ke:
+        LOG.error('No websocket configs found')
+        LOG.exception(ke)
```

Comment:
Having a stack trace for a KeyError seems overly detailed.

maybe just

```python
LOG.error('No websocket configs found ({})'.format(repr(ke)))
```

This would give the line number and the missing key